main/rfid/rfid.c

```diff
@@ -161,18 +174,18 @@ esp_err_t rfid_transceive(
 	rfid_pcd_register_t registers[] = {COM_IRQ_REG, ERROR_REG, FIFO_LEVEL_REG};
 
 	uint16_t attempts_left = 5000;
-	uint8_t output_buffer[1] = {0};
+	uint8_t interrupt_register = 0;
```

Comment:
Is geen array meer. Is volgens mij een probleem

Reply:
This is fine. An array of 1 is equivalent to a single variable, so this is fine